modules/social_features/social_profile/src/Form/SocialProfileSettingsForm.php

```diff
@@ -98,6 +98,41 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ];
     }
 
+    $form['tagging'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Tag settings'),
+      '#open' => TRUE,
+    ];
+
+    $form['tagging']['enable_profile_tagging'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Allow users to tag in profile.'),
+      '#required' => FALSE,
+      '#default_value' => $config->get('enable_profile_tagging'),
+      '#description' => $this->t('Determine whether users are allowed to tag profiles, view tags and filter on tags in profiles.'),
```

Comment:
> Determine whether users are allowed to tag profiles, view tags and filter on tags in profiles.

Don't you mean filter on tags in search?

Also would be great to directly link to the profile tag vocabulary, so a SM / CM can add tags directly. Otherwise they are searching around where to do this.